Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -347,4 +349,25 @@ else if (elementKind.equals(ElementKind.TOP_LEVEL_VARIABLE)) {
       return null;
     }
   }
+
+  @Override
+  public void beforeCompletion(@NotNull final CompletionInitializationContext context) {
```

Comment:
Ok, clear. We do not have the policy to rearrange the code before commit, only to reformat. To switch it off invoke Cmd+Alt+Shift+L, uncheck Rearrange, press Enter. Next time you may use standard dialog-less Cmd+Alt+L.